File: ARealMVP/src/main/java/com/alfish/arealmvp/message/UserLoginResponse.java
package com.alfish.arealmvp.message;

import com.alfish.arealmvp.enums.AuthRejectReason;
import lombok.*;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@ToString
public class UserLoginResponse {

	private String username;
	private String token;
	private AuthRejectReason rejectReason;

}

File: ArealMVP-AR/Assets/Messages/UserLoginResponse.cs
namespace Messages
{
    public class UserLoginResponse
    {
        private string username;
        private string token;
        private string rejectReason;

        public UserLoginResponse()
        {
        }

        public UserLoginResponse(string username, string token, string rejectReason)
        {
            this.username = username;
            this.token = token;
            this.rejectReason = rejectReason;
        }

        public string Username
        {
            get => username;
            set => username = value;
        }

        public string Token
        {
            get => token;
            set => token = value;
        }

        public string RejectReason
        {
            get => rejectReason;
            set => rejectReason = value;
        }
    }
}